pkg/addon/reader_gitee.go

```diff
@@ -100,7 +100,7 @@ func (g *giteeReader) listAddonMeta(dirPath string) ([]Item, error) {
 	return res, nil
 }
 
-// ReadFile read file content from github
+// ReadFile read file content from gitee
 func (g *giteeReader) ReadFile(relativePath string) (content string, err error) {
```

Comment:
Why change this?

Reply:
from the file name `reader_gitee.go` this is to get gitee file, not github

Reply:
ok, you are right :)